projects/stm32_F411RE_nucleo_64_mqtt_mutual_auth_demo/Core/Inc/cellular_config.h

```diff
@@ -0,0 +1,97 @@
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ *
+ * https://www.FreeRTOS.org
+ * https://github.com/FreeRTOS
+ *
+ */
+
+/**
+ * @file cellular_config.h
+ * @brief cellular config options.
+ */
+
+#ifndef __CELLULAR_CONFIG_H__
+#define __CELLULAR_CONFIG_H__
+
+/**************************************************/
+/******* DO NOT CHANGE the following order ********/
+/**************************************************/
+
+/* Include logging header files and define logging macros in the following order:
+ * 1. Include the header file "logging_levels.h".
+ * 2. Define the LIBRARY_LOG_NAME and LIBRARY_LOG_LEVEL macros depending on
+ * the logging configuration for DEMO.
+ * 3. Include the header file "logging_stack.h", if logging is enabled for DEMO.
+ */
+
+#include "logging_levels.h"
+
+/* Logging configuration for the Demo. */
+#ifndef LIBRARY_LOG_NAME
+    #define LIBRARY_LOG_NAME    "CellularLib"
+#endif
+
+#ifndef LIBRARY_LOG_LEVEL
+    #define LIBRARY_LOG_LEVEL    LOG_INFO
+#endif
+
+#include "logging_stack.h"
+
+/************ End of logging configuration ****************/
+
+/* This is a project specific file and is used to override config values defined
+ * in cellular_config_defaults.h. */
+
+/**
+ * Cellular comm interface make use of COM port on computer to communicate with
+ * cellular module on windows simulator, for example "COM5".
+ * #define CELLULAR_COMM_INTERFACE_PORT    "...insert here..."
+ */
+
+/*
+ * Default APN for network registration.
+ * #define CELLULAR_APN                    "...insert here..."
+ */
+
+/*
+ * PDN context id for cellular network.
+ */
+#define CELLULAR_PDN_CONTEXT_ID         ( CELLULAR_PDN_CONTEXT_ID_MIN )
+
+/*
+ * PDN connect timeout for network registration.
+ */
+#define CELLULAR_PDN_CONNECT_TIMEOUT    ( 100000UL )
+
+/*
+ * Overwrite default config for different cellular modules.
+ */
+
+/*
+ * GetHostByName API is not used in the demo. IP address is used to store the hostname.
+ * The value should be longer than the length of democonfigMQTT_BROKER_ENDPOINT in demo_config.h.
+ */
+#define CELLULAR_IP_ADDRESS_MAX_SIZE    ( 64U )
+
+#define CELLULAR_CONFIG_DISABLE_FLOW_CONTROL
```

Comment:
```suggestion
/* STM32 F411RE doesn't support timer, so we need to disable flow control. */
#define CELLULAR_CONFIG_DISABLE_FLOW_CONTROL
```

Reply:
The reason to disable flow control is that the CTS/RTS pins are not connected together. I will add comments in the config.h file.